### DISCUSSION TITLE
Scrutini

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#728] [Richieste di aiuto] [answered]
Buonasera è possibile ora eseguire gli scrutini?
Dato che mi usciva sempre errore

**Comment by luigirotondi2009** [comment]
Salve, nel nostro istituto abbiamo svolto gli scrutini con tutti i nuovi aggiornamenti e funziona tutto benissimo.

**Comment by trinko** [comment, marked as answer]
> Buonasera è possibile ora eseguire gli scrutini? Dato che mi usciva sempre errore

Si, la nuova versione è stata testata per gli scrutini finali. Fai anche gli aggiornamenti successivi alla 1.5.8

**Comment by Davide031** [comment]
Grazie mille

Il dom 16 giu 2024, 20:04 Antonello Dessì ***@***.***> ha
scritto:
